client/blocks/importer/wordpress/upgrade-plan/wordpress-cwvtech-report.json

```diff
@@ -0,0 +1,7 @@
+{
+	"date": "Mar 2024",
+	"technology": "WordPress.com",
+	"goodLCP": 0.8341345685980653,
+	"goodCLS": 0.8036190653856916,
+	"goodFID": 0.99928994923137
```

Comment:
And, related to my [previous comment](https://github.com/Automattic/wp-calypso/pull/90457/files#r1597254247), I think this `goodFID` are the percentage of WPCOM sites with FID lower than 100ms, right? I'm interpreting this based on this PR and [what Google says a good FID is](https://web.dev/articles/fid#what_is_a_good_fid_score).

Reply:
That's correct!